types/topology.go

```diff
@@ -285,6 +285,19 @@ func (t *Topology) GetNodeCmd(name string) string {
 	return ""
 }
 
+func (t *Topology) GetNodeExec(name string) []string {
+	if ndef, ok := t.Nodes[name]; ok {
+		if ndef.GetExec() != nil {
+			return ndef.GetExec()
+		}
+		if t.GetKind(t.GetNodeKind(name)).GetExec() != nil {
+			return t.GetKind(t.GetNodeKind(name)).GetExec()
+		}
+		return t.GetDefaults().GetExec()
+	}
+	return nil
+}
+
```

Comment:
maybe lets do an append from the defaults->kinds->node here?
for example, I might want by default to execute `bash cmd1.sh` on all nodes, then for linux kinds I want to add `bash cmd2.sh` and then for a specific `nodeX` I add `bash cmd3.sh`

In the end, for nodeX the commands will get appended starting from defaults and will comprise the list

- bash cmd1.sh 
- bash cmd2.sh
- bash cmd3.sh